# HG changeset patch
# User William Pitcock <nenolod@atheme.org>
# Date 1293589809 21600
# Node ID c770e450d8d504772b4144ca489f0025d4b61501
# Parent  531718a9e02f0568e5e774d9c05ae764f1a49197
ffaudio: Fix linking order when linked on non-GNU operating systems.

diff -r 531718a9e02f -r c770e450d8d5 configure.ac
--- a/configure.ac	Tue Dec 28 17:43:07 2010 -0500
+++ b/configure.ac	Tue Dec 28 20:30:09 2010 -0600
@@ -843,7 +843,7 @@
 )
 
 if test "x$enable_ffaudio" = "xyes"; then
-    PKG_CHECK_MODULES(FFMPEG, [libavformat >= 52.31.0  libavcodec >= 52.20.0  libavutil >= 49.15.0],
+    PKG_CHECK_MODULES(FFMPEG, [libavcodec >= 52.20.0  libavutil >= 49.15.0  libavformat >= 52.31.0],
                       [have_ffaudio="yes"], [have_ffaudio="no"])
 fi
 
